sfm/ui/test_forms.py

```diff
@@ -163,3 +164,114 @@ def setUp(self):
     def test_valid_from(self):
         form = SeedForm(self.data, seedset=self.seedset.pk)
         self.assertTrue(form.is_valid())
+
+
+class CredentialFlickrFormTest(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.path = "/ui/credentials/flickr/create/"
+        self.user = User.objects.create_superuser(username="test_user",
+                                             email="test_user@test.com",
+                                             password="test_password")
+        self.data = {
+            "name": "test_flickr_credential",
+            "user": self.user.pk,
+            "platform": "flickr",
+            "key":"dummy_key",
+            "secret":"dummy_secret",
+            "date_added": "04/14/2016",
+        }
+
+    def test__form(self):
+        form = CredentialFlickrForm(self.data)
+        form.instance.user = self.user
+        self.assertTrue(form.is_valid())
+        credential = form.save()
+        self.assertTrue(credential.token,
```

Comment:
assertTrue is checking that there is a token and not that it's equal to the string that follows. Use assertEqual instead?